src/components/navigation/TeamSwitcher.tsx

```
@@ -10,12 +10,11 @@ export default function TeamSwitcher() {
   return (
     <div className="relative inline-block text-left">
       <div className="group">
-        <button 
+        <button
           className="px-3 py-2 text-sm font-medium rounded-md bg-gray-100 hover:bg-gray-200"
           aria-haspopup="menu"
           aria-expanded="false"
           aria-controls="team-switcher-menu"
-          role="button"
         >
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)